c_emulator/riscv_sim.cpp

```diff
@@ -186,6 +189,7 @@ static void setup_options(CLI::App &app)
         config_print_rvfi = true;
         config_print_platform = true;
         config_print_step = true;
+        config_print_ptw = true;
```

Comment:
I wonder if we do want to enable this with `--trace-all`. It's probably way more detail than you usually want but if we don't do it it makes the `all` name misleading!

Does anyone have any thoughts?

Reply:
I think trace all should always equals to trace everything :(